pkg/api/licenses.go

```diff
@@ -1152,3 +1152,60 @@ func ExportLicenses(c *gin.Context) {
 	c.Header("Content-Disposition", fmt.Sprintf("attachment; filename=%s", fileName))
 	c.JSON(http.StatusOK, &licenses)
 }
+
+// GetAllLicensePreviews retrieves a list of topics and types of all obligations
```

Comment:
Is the comment correct? We are only fetching Licenses, not obligations.

Reply:
Yes, my bad